src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorService = new ColorSupplier();
```

Comment:
Better not not have static when we declare services we rely on
```suggestion
    private final Random random;
    private final ColorSupplier colorService;
```

and also please use initialization with constructor and create instances only in entry-point class.
Please recheck all services to use proper constructor

Reply:
Need explanation